report/renderer/util.js

```diff
@@ -150,6 +150,12 @@ class Util {
             }
           }
         }
+
+        if (audit.details.type === 'opportunity' || audit.details.type === 'table') {
+          // Attach table/opportunity items with entity information.
+          Util.getEntityClassifiedTableItems(audit.id, result.entityClassification,
```

Comment:
Why not pass in the entire audit? Would have cleared up some confusion I had referenced in my other comment.

Reply:
Merge the params `id`, `headings`, and `items`? The function will have to accept a more generic type and do same type checks again to keep TS happy, I think — other than that no downside, should be doable.

Reply:
This is now resolved, we pass the complete audit object.